resource/decker/support/plugins/decker/ui-anchors.js

```diff
@@ -19,7 +19,6 @@
 
 class DeckerPlugins {
```

Comment:
Please rename to UIAnchorsPlugin or something more meaningful

Reply:
done